tools/cli_commands/cost_report/openshift.py

```diff
@@ -0,0 +1,176 @@
+from collections import defaultdict
+from collections.abc import Iterable, Mapping
+from decimal import Decimal
+from typing import Self, Tuple
+
+from sretoolbox.utils import threaded
+
+from reconcile.typed_queries.app_interface_vault_settings import (
+    get_app_interface_vault_settings,
+)
+from reconcile.typed_queries.cost_report.app_names import App, get_app_names
+from reconcile.typed_queries.cost_report.cost_namespaces import (
+    CostNamespace,
+    get_cost_namespaces,
+)
+from reconcile.typed_queries.cost_report.settings import get_cost_report_settings
+from reconcile.utils import gql
+from reconcile.utils.secret_reader import create_secret_reader
+from tools.cli_commands.cost_report.cost_management_api import CostManagementApi
+from tools.cli_commands.cost_report.model import ChildAppReport, Report, ReportItem
+from tools.cli_commands.cost_report.response import OpenShiftReportCostResponse
+from tools.cli_commands.cost_report.util import process_reports
+from tools.cli_commands.cost_report.view import render_openshift_cost_report
+
+THREAD_POOL_SIZE = 10
```

Comment:
nit: The thread pool size should be configurable via `__init__`

Reply:
the way we call this command has no input options, so I didn't make it configurable at cli level, only added init params at https://github.com/app-sre/qontract-reconcile/pull/4305/commits/24d6a672439016d0ee8ea5ccf30bfa58079d6336